### PR TITLE
docs: add windows installation

### LINKED ISSUE #1179: Adding installation guide for windows user, using binary distribution (instead of docker)
**Please describe the feature you want**

Many Windows Tabby users are not aware of our Windows distribution. It would be beneficial to have a Windows-specific installation tutorial on our website at https://tabby.tabbyml.com/docs/installation/.

---
Please reply with a 👍 if you want this feature.

### PR DESCRIPTION
**Pull request by wwayne**
fix #1179

### CODE REVIEW

File: website/docs/installation/windows/index.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Running Tabby on Windows without WSL or Docker.
+
+## 1. Find the Windows release
+* Go to the Tabby release page: https://github.com/TabbyML/tabby/releases
+* Click on the **Assets** dropdown for a specific release to find the Windows executable (exe) files.
+![Windows release](./win-release.png)
+
+## 2. Download the release
+* If you are using a CPU-only system, download the **tabby_x86_64-windows-msvc.exe file**.
+* If you are using a GPU-enabled system, download the **tabby_x86_64-windows-msvc-cuda{VERSION}.exe** file, replacing {VERSION} with the appropriate CUDA version for your system.
```
Comment (wsxiaoys) [inline website/docs/installation/windows/index.mdx:15]: use a concrete version number (e.g 11.7) as an example.

File: website/docs/installation/windows/index.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Check your local CUDA version by running the following command in a command prompt or PowerShell window:
+```
+nvcc --version
+```
+
+## 3. Running Tabby locally
+Open a command prompt or PowerShell window in the directory where you downloaded the Tabby executable. Run the following command:
+```
+.\tabby_x86_64-windows-msvc-cuda{VERSION}.exe serve --device cuda
+```
```
Comment (wsxiaoys) [inline website/docs/installation/windows/index.mdx:28]: add more steps to verify installation success (e.g open swagger page and run some inference)

File: website/docs/installation/windows/index.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 3
+---
+
+# Windows
+Running Tabby on Windows without WSL or Docker.
```
Comment (wsxiaoys) [inline website/docs/installation/windows/index.mdx:6]:
```suggestion
Running Tabby on Windows using Tabby's exe distribution
```

**Comment by wwayne** [comment]
updated based on the feedbacks

File: website/docs/installation/windows/index.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Click on the **Assets** dropdown for a specific release to find the Windows executable (exe) files.
+![Windows release](./win-release.png)
+
+## 2. Download the release
+* If you are using a CPU-only system, download the **tabby_x86_64-windows-msvc.exe file**.
+* If you are using a GPU-enabled system, download the **tabby_x86_64-windows-msvc-cuda117.exe** file, In this example, we assume you are using CUDA 11.7.
+
+**Tips:**
+* Download the CUDA Toolkit from Nvidia: https://developer.nvidia.com/cuda-toolkit
+* Check your local CUDA version by running the following command in a command prompt or PowerShell window:
```
Comment (gyxlucy) [inline website/docs/installation/windows/index.mdx:19]: What version is needed? Is there a minimum requirement?

**Review by gyxlucy — CHANGES_REQUESTED** [review]
One question for clarification

File: website/docs/installation/windows/index.mdx (hunk 4 above)
Comment (wwayne) [reply website/docs/installation/windows/index.mdx:20]: Looks both 11 and 12 are suppoted?

This is CUDA download page
屏幕截图 2024-03-09 100119

I add a new tip to the list
`Ensure that you have CUDA version 11 or higher installed.`

Comment (wwayne) [reply website/docs/installation/windows/index.mdx:20]: @icycodes if you can help to double confirm? thanks

Comment (wsxiaoys) [reply website/docs/installation/windows/index.mdx:20]: cuda 11 is good enough.

**Review by gyxlucy — APPROVED** [review]

**Pull request merged**